snforge_std/src/cheatcodes/execution_info.cairo

```diff
@@ -1,36 +1,36 @@
 use starknet::{ContractAddress, testing::cheatcode, contract_address_const};
-use starknet::info::v2::ResourceBounds;
+use starknet::ResourcesBounds;
 use snforge_std::cheatcodes::CheatSpan;
 use super::super::_cheatcode::handle_cheatcode;
 
-mod caller_address;
-mod block_number;
-mod block_timestamp;
-mod sequencer_address;
-mod version;
-mod max_fee;
-mod signature;
-mod transaction_hash;
-mod chain_id;
-mod nonce;
-mod resource_bounds;
-mod tip;
-mod paymaster_data;
-mod nonce_data_availability_mode;
-mod fee_data_availability_mode;
-mod account_deployment_data;
-mod account_contract_address;
+pub mod caller_address;
+pub mod block_number;
+pub mod block_timestamp;
+pub mod sequencer_address;
+pub mod version;
+pub mod max_fee;
+pub mod signature;
+pub mod transaction_hash;
+pub mod chain_id;
+pub mod nonce;
+pub mod resource_bounds;
+pub mod tip;
+pub mod paymaster_data;
+pub mod nonce_data_availability_mode;
+pub mod fee_data_availability_mode;
+pub mod account_deployment_data;
+pub mod account_contract_address;
 
 
 #[derive(Serde, Drop, Copy)]
-struct CheatArguments<T> {
-    value: T,
-    span: CheatSpan,
-    target: ContractAddress,
+pub struct CheatArguments<T> {
+    pub value: T,
+    pub span: CheatSpan,
+    pub target: ContractAddress,
 }
 
 #[derive(Serde, Drop, Copy)]
-enum Operation<T> {
+pub enum Operation<T> {
```

Comment:
```suggestion
enum Operation<T> {
```

Reply:
I suppose that it should stay public:
https://github.com/foundry-rs/starknet-foundry/blob/master/crates/forge/tests/integration/cheat_execution_info.rs#L151-L155

It also shows that `CheatArguments` should be public